src/FEValues/CellValues.jl

```diff
@@ -127,4 +127,110 @@ function Base.show(io::IO, d::MIME"text/plain", cv::CellValues)
     print(io, "\n Function interpolation: "); show(io, d, ip_fun)
     print(io, "\nGeometric interpolation: "); 
     sdim === nothing ? show(io, d, ip_geo) : show(io, d, ip_geo^sdim)
-end
\ No newline at end of file
+end
+
+"""
+    MultiCellValues([::Type{T},] quad_rule::QuadratureRule, func_interpols::NamedTuple, [geom_interpol::Interpolation])
```

Comment:
Maybe `CellMultiValues` for the name, because we have multiple values (in contrast to multiple cells)?